django/core/static/css/custom.css

```diff
@@ -69,6 +69,44 @@ h3 {
     height: 100%;
 }
 
+.accordian {
```

Comment:
accordion

Reply:
thanks, have corrected this throughout the project